site/content/presentations-material.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Presentations Material"
+menu = "main"
+weight = 620
```

Comment:
we already have a page at this position in the menu (`contributing.md`).
I would suggest to position this page between `Architecture` and `Developing Hono` which would result in a weight of  e.g. 590

Reply:
I think we should also rename it to something like  `Presentations et al.`

Reply:
What's the way for defining the position ? I'm taking a look at `config.toml` (is it the right file ?) and I see `Architecture` but not `Developing Hono` :(

Reply:
You can either have explicit entries in the menu in order to support multiple pages under that entry (like we do for Architecture and Deplyoment etc)  or you can define a single page and use the front matter to define it as a (single page) entry in menu (like we do here). The position is then determined by the weight indicated in the front matter.

Reply:
Ok ! I didn't understood that `weight` was the right parameter for that :-) 
Regarding the title maybe just `Presentations` or something different ? Using Latin sounds not to good to me :-)